### PR TITLE
Move app/labor/follow_checker.rb to service

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

This PR moves `app/labor/follow_checker.rb` to `app/services/check_cached_follows.rb` and updates the specs and code accordingly.

## Related Tickets & Documents

https://github.com/forem/InternalProjectPlanning/issues/271

## Added tests?

- [X] Updated existing ones

## Added to documentation?

- [X] No documentation needed

### CODE REVIEW

File: app/services/check_cached_follows.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class CheckCachedFollows
```
Comment (rhymes) [inline app/services/check_cached_follows.rb:1]: I vote for namespacing this to `Follows::` - I think eventually our `app/services` folder should be grouped by section or functionality in someway, with less classes in the top level

**Comment by citizen428** [comment]
@rhymes Moved it.

**Review by rhymes — APPROVED** [review]
LGTM

**Review by juliannatetreault — APPROVED** [review]
Thanks for taking the time to move this! 😄

**Review by jacobherrington — APPROVED** [review]

**Pull request merged**